samples/openapi3/client/petstore/python-nextgen-aiohttp/petstore_api/api/pet_api.py

```diff
@@ -460,7 +460,7 @@ def find_pets_by_status_with_http_info(self, status : Annotated[conlist(StrictSt
         # process the query parameters
         _query_params = []
         if _params.get('status') is not None:  # noqa: E501
-            _query_params.append(('status', _params['status']))
```

Comment:
I'm guessing this is unrelated to your changes? Might want to rebase and/or merge and regenerate samples?

Reply:
@rodrigoma3 let me fix this in master and then you can rebase your changes

Reply:
This actually appeared after I rebase and regenerate samples.

Reply:
I think whoever requested the last PR didn't regenerate samples.

Reply:
@rodrigoma3 you can rebase now. I've update samples in the master

Reply:
@borsch 
done